packages/react-components/react-theme/src/types.ts

```diff
@@ -42,6 +42,7 @@ export type ColorTokens = {
   colorNeutralForegroundInvertedPressed: string;
   colorNeutralForegroundInvertedSelected: string;
   colorNeutralForegroundOnBrand: string;
+  colorNeutralForegroundInvertedStatic: string;
```

Comment:
The new tokens must be also added to https://github.com/microsoft/fluentui/blob/eeed967f604dc4f6a4b6b32b6347c0ed9360294c/packages/react-components/react-theme-sass/sass/colorTokens.scss.

There is a test which is supposed to catch this, I have no idea why it has not failed. I will investigate in #24161